### DISCUSSION TITLE
Windows defender swears by trojans

### DISCUSSION THREAD

**Discussion by AndyGolos** [pyinstaller/pyinstaller#7564] [Help]
Hi team!

I've created .exe file using latest version of pyinstaller (5.10.0) and after sharing this exe file to another user windows defender starts swearing on trojans in this file on his PC. Could you please proof that this issue is on windows defender side? Or maybe you can suggest me what to do?

Python version 3.11.3

![image](https://user-images.githubusercontent.com/17516241/231770579-857cd39c-a839-4d2c-bc61-792ad7da45e9.png)

**Comment by bwoodsend** [comment]
Waiting a few days normally is enough for Windows Defender to change its mind. Report it to Microsoft if it doesn't - it's their bug.

**Reply by AndyGolos** [reply]
Will issue disappear if i will use previous version of pyinstaller? 5.9.0?

**Reply by bwoodsend** [reply]
It's a psuedo random process. It might fix it now or it might not. It might also break it later on. There's no point randomly rearranging things. Either report the false positive to the AV vendor or ignore it.

**Reply by rokm** [reply]
You can always rebuild the bootloader yourself if you don't trust the prebuilt ones that are shipped with PyInstaller wheels and checked into the repository. Then if AV flags your executable as virus, you'll be more confident that it is a false positive...

Since different toolchains produce slightly different results, rebuilding bootloader yourself might (or might not) also help with AV false positives.
